AppInspector/rules/default/cryptography/certificate.json

```diff
@@ -32,17 +28,13 @@
     "id": "AI006300",
     "description": "Cryptography: x.509 Certificate Use",
     "applies_to": [ "c", "cpp", "csharp" ],
-    "tags": [
-      "Cryptography.X509Certificates.Use"
-    ],
+    "tags":[ "Cryptography.X509Certificates.Use" ],
     "severity": "moderate",
     "patterns": [
       {
         "pattern": "_CERT|CERT_",
```

Comment:
Maybe `_?CERT_?` instead?

Reply:
This would capture any time the 4 letters are present because you can't enforce there's one on either side.

Reply:
Sounds good.